modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/configuration/schema/PageMemoryStorageEngineConfigurationSchema.java

```diff
@@ -35,6 +40,14 @@ public class PageMemoryStorageEngineConfigurationSchema {
     /** Name of the default data region. */
     public static final String DEFAULT_DATA_REGION_NAME = "default";
 
+    /** Page size in bytes. */
+    @Immutable
+    @Min(1024)
+    @Max(16 * 1024)
+    @PowerOfTwo
+    @Value(hasDefault = true)
+    public int pageSize = 16 * 1024;
```

Comment:
Ok, what was the point of having new isPow2 for longs if you only check integer values?

Reply:
Discussed personally, this issue is minor.